### PR TITLE
Added pagination functionality

### PR DESCRIPTION
**Pull request by mikeengland**

### CODE REVIEW

File: fireant/slicer/pagination.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+class Paginator(object):
+    def __init__(self, limit=0, offset=0, order=()):
+        """
+        Class for keeping track of pagination parameters
+
+        Args:
```
Comment (x8lucas8x) [inline fireant/slicer/pagination.py]: I have nothing against google doc style, but mixing styles in a project does not feel right to me.

File: fireant/slicer/queries.py [hunk 2]
```diff
@@ -310,6 +343,18 @@ def _add_sorting(query, dimensions):
...
 
     @staticmethod
+    def _add_pagination(query, pagination, metrics, dimensions, references):
+        """ Add offset, limit and order pagination to the query """
+        query = query[pagination.offset: pagination.limit]
+        query_objects = utils.merge_dicts(metrics, dimensions, references)
+
+        for key, order in pagination.order:
+            field = query_objects.get(key)
+            query = query.orderby(field, order=order)
```
Comment (x8lucas8x) [inline fireant/slicer/queries.py]: Is there a way to order by without paginating?

File: fireant/slicer/transformers/__init__.py [hunk 3]
```diff
@@ -21,27 +21,40 @@
...
+BAR_CHART = 'bar_chart'
+AREA_CHART = 'area_chart'
+AREA_PERCENTAGE_CHART = 'area_percentage_chart'
+COLUMN_CHART = 'column_chart'
+STACKED_COLUMN_CHART = 'stacked_column_chart'
+STACKED_BAR_CHART = 'stacked_bar_chart'
+PIE_CHART = 'pie_chart'
+
 BUNDLES = {
     'notebooks': {
```
Comment (x8lucas8x) [inline fireant/slicer/transformers/__init__.py]: Thanks for removing the hardcoded types of chart bellow.

File: fireant/slicer/pagination.py (hunk 1 above)
Comment (mikeengland) [inline fireant/slicer/pagination.py]: Makes sense, my PyCharm Fireant project was set to that for some reason!

File: fireant/slicer/queries.py (hunk 2 above)
Comment (mikeengland) [inline fireant/slicer/queries.py]: Right now there is no way to do it. Maybe we should split these out into two different objects that can get passed in. The default behaviour as of master right now is to just sort the dimensions. What do you think?

**Comment by mikeengland** [comment]
I am going to refactor the Paginator object into a Paginator object, but also add an Order object as we will probably need that too.

**Pull request merged**